docs/linter_settings.rst

```diff
@@ -42,7 +42,7 @@ The value must be a string, corresponding to a valid directory path.
         "chdir": "${project}"
     }
 
-With the above example, the linter will get invoked from the ``${project}`` directory (see :ref:`Setting Tokens <settings-tokens>` for more info on using tokens).
+With the above example, the linter will get invoked from the ``${project}`` directory (see :ref:`Settings Expansion <settings-expansion>` for more info on using variables).
```

Comment:
project -> folder

Reply:
The example still lists `${project}`, so the description is correct. I suspect we'd want this to just show the default, though, don't we?

Reply:
Ah well, it's just that end-users get confused bc they relate ${project} to be the base dir of their app. But it's a Sublime name here referring to the sublime project file. End-users usually are looking for ${folder}.